app/controllers/items_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ItemsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @list = List.find(params[:list_id])
```

Comment:
Favor user-scoped queries where possible for extra precaution against exposing user data to anyone who fiddles with request/URL id parameters. Line 5 would become:

```ruby
@list = current_user.lists.find(params[:list_id])
```